src/main/js/controls/PairedResult/PairedResult.js

```diff
@@ -188,10 +191,101 @@ class PairedResult extends GraphContent {
      * @inheritDoc
      */
     resize(graph) {
+        const regionItm = [];
+        let localRegionSameFlag = false;
+
         if (utils.notEmpty(this.dataTarget.regions)) {
-            // If graph has more than 1 content then we dont load the regions
+            const regionList = this.dataTarget.regions;
+            const values = this.dataTarget.values;
+
+            //check if all region are there with respect to value
+            if (graph.config.isPairedDataProper === true) {
+                for (let i = 0; i < values.length; i++) {
+                    // eslint-disable-next-line max-depth
+                    for (const key in values[i]) {
+                        // eslint-disable-next-line max-depth
+                        if (!regionList.hasOwnProperty(key)) {
+                            graph.config.isPairedDataProper = false;
+                            break;
+                        }
+                    }
+                    // eslint-disable-next-line max-depth
+                    if (graph.config.isPairedDataProper === false) {
+                        break;
+                    }
+                }
+            }
+
+            // Internal region comparision to check if the internal region is same
+            if (graph.config.isPairedDataProper === true) {
+                for (const key in regionList) {
+                    const region = regionList[key];
+                    // eslint-disable-next-line max-depth
+                    for (let i = 0; i < region.length; i++) {
+                        // eslint-disable-next-line max-depth
+                        if (regionItm.length === 0) {
+                            regionItm.push(region[i]);
+                        }
+                        // eslint-disable-next-line max-depth
+                        if (regionItm.length > 0) {
+                            // eslint-disable-next-line max-depth
+                            if (
+                                regionItm[0].start === region[i].start &&
```

Comment:
Might probably use https://www.npmjs.com/package/deep-equal package for comparison of objects. Think there should be one in the helpers as well.

Reply:
ok i will use it for the object comparison

Reply:
I am not for this for 2 reasons: 
* If we deep equal on an input object then in future you cannot include an object property without breaking this functionality, if it doesnt apply to condition here. For instance, if we decide a `label` or `onClick` property. 
* Pulling in additional dependencies

Reply:
so you want me not to use deep-equal

Reply:
Don't we have a comparator sought of thing in our helpers @abhijit945, which we can use?